Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -13,6 +13,8 @@ import Foundation
 /// There are two different types of services you can retrieve a server for: XPC Services and XPC Mach services. If you're uncertain which type of service you're
 /// using, it's likely an XPC Service.
 ///
+/// Anonymous servers can also be created which do not correspond to a service.
```

Comment:
This is a bit jargony IMO. I'm not sure "a server without a service" would make sense unless people were already in-the-know.

Perhaps:

```suggestion
/// Anonymous servers can also be created, which do not correspond to a Mach or XPC service. These can be useful for testing, or to aid in development by removing the need to debug across multiple processes.
```

Reply:
NVM, you cover use cases below, but you get the idea

Reply:
Good call, will incorporate the first portion of this.